docs/guides/languages.md

```diff
@@ -1,15 +1,21 @@
-Languages
-=========
+# Languages
+Multiple language support allows for users of non-English locales to natively interact with the Video.js player.
 
-Multiple language support allows for users of non-English locales to natively interact with the displayed player. Video.js will compile multiple language files (see below) and instantiate with a global dictionary of language key/value support. Video.js player instances can be created with per-player language support that amends/overrides these default values. Player instances can also hard-set default languages to values other than English as of version 4.7.
+For an up-to-date list of the languages Video.js supports, see the [languages folder (`lang`)][lang-supported]. These JSON files are converted to JavaScript during the Video.js build process.
 
-Creating the Language File
---------------------------
-Video.js uses key/value object dictionaries in JSON form.
+## Using Video.js Languages
+Video.js ships with multiple translations (in `dist/lang/`) in JavaScript files. Each of these files can be included in a web page to provide support for that language in _all_ Video.js players:
 
-An English lang file is at [/lang/en.json](https://github.com/videojs/video.js/tree/master/lang/en.json) which should be used as a template for new files. This will be kept up to date with strings in the core player that need localizations.
+```
+<script src="//example.com/path/to/video.min.js"></script>
+<script src="//example.com/path/to/lang/es.js"></script>
+```
+
+## Contributing to Video.js Translations
+We welcome new translations and improvements to existing ones! Please see the [contributing document](../../CONTRIBUTING.md) to get started contributing to Video.js and continue reading for specifics on how to contribute to translations of Video.js...
```

Comment:
I think it would be good to have a specific entry for language contributions as I think that it might be most difficult for language contributors to contribute.